src/main/java/com/google/sps/servlets/CareerQuizServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.CareerQuestionDatabase;
+import com.google.sps.data.CareerQuestionAndChoices;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+
+@Singleton
+/** Responds with a JSON string containing questions and answers for the career quiz*/
+public class CareerQuizServlet extends HttpServlet {
+  private CareerQuestionDatabase careerQuestionDatabase ;
```

Comment:
I think you can just use [field injection](https://www.tutorialspoint.com/guice/guice_field_injection.htm) on this member variable here:
`@Inject private final CareerQuestionDatabase careerQuestionDatabase;`

Otherwise, with this pattern, you wouldn't need Guice at all. Because you can just do:
```
private CareerQuestionDatabase careerQuestionDatabase = new CareerQuestionDatabase(datastore);
```
And then have the setter. Or you can move this call:
```
ArrayList<CareerQuestionAndChoices> questionsAndChoices = 
        this.careerQuestionDatabase.getQuestionsAndChoices()
```
to a new method and mock that method.

I left a comment in the test to use guice to inject a mock